clients/drcachesim/tracer/raw2trace.cpp

```diff
@@ -1614,6 +1632,13 @@ raw2trace_t::do_conversion()
     VPRINT(1, "Kernel instr count " UINT64_FORMAT_STRING "\n", kernel_instr_count_);
     VPRINT(1, "System call PT traces decoded " UINT64_FORMAT_STRING "\n",
            syscall_traces_decoded_);
+    VPRINT(1, "System call PT traces decode failed " UINT64_FORMAT_STRING "\n",
```

Comment:
This could be mis-interpreted as individual instruction decode calls that failed (some of which were re-tried): is there another name: maybe s/decode/conversion/